packages/core/test/unit/process/4-mutation-test-executor.spec.ts

```diff
@@ -401,4 +405,22 @@ describe(MutationTestExecutor.name, () => {
     expect(checker.check).not.called;
     expect(actualResults).empty;
   });
+
+  it('should short circuit when no tests found and allowEmpty is enabled', async () => {
```

Comment:
Please also add a test to not short circuit when no tests are found and `allowEmpty` is disabled.

Reply:
Hi @nicojs 
I added a test when short circuit should be skipped if no tests are found and allowEmpty is disabled.